### PR TITLE
graphviz: for stopping errors with --with-bindings

### PR DESCRIPTION
**Pull request by s172262**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
----
This PR is reopening https://github.com/Homebrew/homebrew-core/pull/19128.

### CODE REVIEW

File: Formula/graphviz.rb [hunk 1]
```diff
@@ -58,6 +58,10 @@ def install
     inreplace "lib/sfio/features/sfio", "lib qfrexp\nlib qldexp\n", ""
 
     if build.with? "bindings"
+      # The following setting makes to find "python.h".
+      # https://github.com/Homebrew/homebrew-core/issues/19110
+      ENV["PYTHON_INCLUDES"] = "-I" << `python -c "from __future__ import print_function;import sysconfig; print (sysconfig.get_path('include'))"`.chomp
```
Comment (ilovezfs) [inline Formula/graphviz.rb]: Something is wrong here if it cannot find it automatically. This shouldn't be necessary.

**Review by ilovezfs — DISMISSED** [review]

File: Formula/graphviz.rb (hunk 1 above)
Comment (s172262) [reply Formula/graphviz.rb]: Without Homebrew's `python`:
```
$ python -c "from __future__ import print_function;import sysconfig; print (sysconfig.get_path('include'))"
/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
```
I understand this is too bad.

**Comment by s172262** [comment]
I hasten to repair PHP's errors.

File: Formula/graphviz.rb [hunk 2]
```diff
@@ -71,6 +71,7 @@ def install
       --prefix=#{prefix}
       --without-qt
       --with-quartz
+      --enable-php=no
```
Comment (ilovezfs) [inline Formula/graphviz.rb]: does `--disable-php` work instead?

Comment (s172262) [reply Formula/graphviz.rb]: Ther is no different `--enable-php=no` and `--disable-php`.
```
==> ./configure --disable-debug --disable-dependency-tracking --prefix=/usr/local/Cellar/graphviz/2.40.1 --without-qt --with-quartz --enable-php=no --with-gts --without-rsvg
```
```
----------------------------------------------------------------

graphviz-2.40.1 will be compiled with the following:

options:
  cgraph:        Yes (always enabled)
  digcola:       Yes
  expat:         Yes
  fontconfig:    Yes
  freetype:      Yes
  glut:          Yes
  ann:           No (ANN library not available)
  gts:           Yes
  ipsepcola:     No (disabled by default - C++ portability issues)
  ltdl:          Yes
  ortho:         Yes
  sfdp:          Yes
  shared:        Yes
  static:        No (disabled by default)
  qt:            No (disabled)
  x:             Yes

commands:
  dot:           Yes (always enabled)
  neato:         Yes (always enabled)
  fdp:           Yes (always enabled)
  circo:         Yes (always enabled)
  twopi:         Yes (always enabled)
  gvpr:          Yes (always enabled)
  gvmap:         Yes (always enabled)
  lefty:         Yes
  smyrna:        No (disabled by default - experimental)
  gvedit:        No (disabled)

plugin libraries:
  dot_layout:    Yes (always enabled)
  neato_layout:  Yes (always enabled)
  core:          Yes (always enabled)
  devil:         No (missing library)
  gd:            Yes
  gdiplus:       No (disabled by default - Windows only)
  gdk:           
  gdk_pixbuf:    No (gdk_pixbuf library not available)
  ghostscript:   Yes
  glitz:         No (disabled by default - incomplete)
  gtk:           No (gtk library not available)
  lasi:          No (lasi library not available)
  ming:          No (disabled by default - incomplete)
  pangocairo:    Yes
  poppler:       No (poppler library not available)
  quartz:        Yes
  rsvg:          No (disabled)
  visio:         No (disabled by default - experimental)
  webp:          No (disabled by default - experimental)
  xlib:          Yes

language extensions:
  gv_sharp:      No (mcs not available)
  gv_go:         No (disabled by default - experimental)
  gv_guile:      No (guile not available)
  gv_io:         No (disabled by default - no swig support yet)
  gv_java:       Yes
  gv_lua:        No (lua not available)
  gv_ocaml:      No (ocaml not available)
  gv_perl:       Yes
  gv_php:        No (disabled)
  gv_python:     Yes
  gv_python23:   No (disabled by default - for multiversion installs)
  gv_python24:   No (disabled by default - for multiversion installs)
  gv_python25:   No (disabled by default - for multiversion installs)
  gv_python26:   No (disabled by default - for multiversion installs)
  gv_python27:   No (disabled by default - for multiversion installs)
  gv_python34:   No (disabled by default - for multiversion installs)
  gv_R:          No (R not available)
  gv_ruby:       Yes
  gv_tcl:        No (tcl not available)

  tcldot:        No (tcl not available)
  tclpathplan:   No (tcl not available)
  gdtclft:       No (tcl not available)

Testing utilities:
  criterion:     No (Criterion unit testing framework not installed)
```

```
==> ./configure --disable-debug --disable-dependency-tracking --prefix=/usr/local/Cellar/graphviz/2.40.1 --without-qt --with-quartz --disable-php --with-gts
```
```
----------------------------------------------------------------

graphviz-2.40.1 will be compiled with the following:

options:
  cgraph:        Yes (always enabled)
  digcola:       Yes
  expat:         Yes
  fontconfig:    Yes
  freetype:      Yes
  glut:          Yes
  ann:           No (ANN library not available)
  gts:           Yes
  ipsepcola:     No (disabled by default - C++ portability issues)
  ltdl:          Yes
  ortho:         Yes
  sfdp:          Yes
  shared:        Yes
  static:        No (disabled by default)
  qt:            No (disabled)
  x:             Yes

commands:
  dot:           Yes (always enabled)
  neato:         Yes (always enabled)
  fdp:           Yes (always enabled)
  circo:         Yes (always enabled)
  twopi:         Yes (always enabled)
  gvpr:          Yes (always enabled)
  gvmap:         Yes (always enabled)
  lefty:         Yes
  smyrna:        No (disabled by default - experimental)
  gvedit:        No (disabled)

plugin libraries:
  dot_layout:    Yes (always enabled)
  neato_layout:  Yes (always enabled)
  core:          Yes (always enabled)
  devil:         No (missing library)
  gd:            Yes
  gdiplus:       No (disabled by default - Windows only)
  gdk:           
  gdk_pixbuf:    Yes
  ghostscript:   Yes
  glitz:         No (disabled by default - incomplete)
  gtk:           No (gtk library not available)
  lasi:          No (lasi library not available)
  ming:          No (disabled by default - incomplete)
  pangocairo:    Yes
  poppler:       No (poppler library not available)
  quartz:        Yes
  rsvg:          Yes
  visio:         No (disabled by default - experimental)
  webp:          No (disabled by default - experimental)
  xlib:          Yes

language extensions:
  gv_sharp:      No (mcs not available)
  gv_go:         No (disabled by default - experimental)
  gv_guile:      No (guile not available)
  gv_io:         No (disabled by default - no swig support yet)
  gv_java:       Yes
  gv_lua:        No (lua not available)
  gv_ocaml:      No (ocaml not available)
  gv_perl:       Yes
  gv_php:        No (disabled)
  gv_python:     Yes
  gv_python23:   No (disabled by default - for multiversion installs)
  gv_python24:   No (disabled by default - for multiversion installs)
  gv_python25:   No (disabled by default - for multiversion installs)
  gv_python26:   No (disabled by default - for multiversion installs)
  gv_python27:   No (disabled by default - for multiversion installs)
  gv_python34:   No (disabled by default - for multiversion installs)
  gv_R:          No (R not available)
  gv_ruby:       Yes
  gv_tcl:        No (tcl not available)

  tcldot:        No (tcl not available)
  tclpathplan:   No (tcl not available)
  gdtclft:       No (tcl not available)

Testing utilities:
  criterion:     No (Criterion unit testing framework not installed)
```

**Comment by ilovezfs** [comment]
Thanks @s172262!

**Pull request merged**